cpp/daal/src/algorithms/linear_model/linear_model_train_normeq_finalize_impl.i

```diff
@@ -47,6 +49,15 @@ template <typename algorithmFPType, CpuType cpu>
 Status FinalizeKernel<algorithmFPType, cpu>::compute(const NumericTable & xtxTable, const NumericTable & xtyTable, NumericTable & xtxFinalTable,
                                                      NumericTable & xtyFinalTable, NumericTable & betaTable, bool interceptFlag,
                                                      const KernelHelperIface<algorithmFPType, cpu> & helper)
+{
+    return FinalizeKernel<algorithmFPType, cpu>::compute(xtxTable, xtyTable, xtxFinalTable, xtyFinalTable, betaTable, interceptFlag, helper, nullptr);
+}
+
+template <typename algorithmFPType, CpuType cpu>
+Status FinalizeKernel<algorithmFPType, cpu>::compute(const NumericTable & xtxTable, const NumericTable & xtyTable, NumericTable & xtxFinalTable,
+                                                     NumericTable & xtyFinalTable, NumericTable & betaTable, bool interceptFlag,
+                                                     const KernelHelperIface<algorithmFPType, cpu> & helper,
+                                                     const HyperparameterType * hyperparameter)
```

Comment:
Hyperparameter is passed into compute(), but not used inside it?

Reply:
This variant of function is just for the sake of bw compatibility. So that even if something depends on this symbol it will work correctly. In comparison having methods with default argument values will result in having unresolved symbol

Reply:
Thanks!